### PR TITLE
Isolate native and group variant data to a single file each

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@nrwahl2 , here's yet another low-priority review, last one for this week :)

File: lib/pengine/group.c [hunk 1]
```diff
@@ -23,6 +23,56 @@
 #define VARIANT_GROUP 1
 #include "./variant.h"
 
+/*!
+ * \internal
+ * \brief Check whether a group flag is set
+ *
+ * \param[in] group  Group resource to check
+ * \param[in] flags  Flag or flags to check
```
Comment (nrwahl2) [inline lib/pengine/group.c:52]: might specify the flag type here and below, but the use case is so limited (and deprecated) it doesn't really matter

**Review by nrwahl2 — COMMENTED** [review]
Looks good

File: lib/pengine/group.c [hunk 2]
```diff
@@ -23,6 +23,56 @@
...
+    // We don't actually need the null check but it speeds up the common case
+    if ((value_s == NULL) || (crm_str_to_boolean(value_s, &value) < 0)
+        || (value != 0)) {
+        group_variant_data_t *group_data = NULL;
+
+        get_group_variant_data(group_data, group);
+        group_data->flags |= flag;
+
+    } else {
+        pe_warn_once(pe_wo_group_order,
```
[the remark this answers is not in the document]
Comment (kgaillot) [reply lib/pengine/group.c:90]: Whoops, I overlooked that

[the remark this answers is not in the document]
Comment (kgaillot) [reply lib/pengine/group.c]: updated commit

**Review by nrwahl2 — APPROVED** [review]

**Pull request merged**